src/main/java/au/org/emii/ncdfgenerator/CreateWritable.java

```diff
@@ -2,29 +2,44 @@
 package au.org.emii.ncdfgenerator;
```

Comment:
Actually, looks like `ncdf` is used a bit too: https://www.google.com.au/search?q=ncdf&oq=ncdf&aqs=chrome..69i57j69i60l5.650j0j7&sourceid=chrome&es_sm=91&ie=UTF-8

I'm not too hung up on `ncdf` vs `netcdf` (although I think the latter is more descriptive).
The more important thing to me is that it's consistent within this project, and with the project name.